frontend/src/components/editor/DocumentView.tsx

```diff
@@ -1,13 +1,14 @@
+import { Backdrop, Box, CircularProgress, Paper } from "@mui/material";
+import { useWindowWidth } from "@react-hook/window-size";
 import { useSelector } from "react-redux";
-import { selectEditor } from "../../store/editorSlice";
 import Resizable from "react-resizable-layout";
-import { useWindowWidth } from "@react-hook/window-size";
+import { ScrollSync, ScrollSyncPane } from "react-scroll-sync";
+import { EditorModeType, selectEditor } from "../../store/editorSlice";
 import Editor from "./Editor";
-import { Backdrop, Box, CircularProgress, Paper } from "@mui/material";
+import EditorBottomBar, { BOTTOM_BAR_HEIGHT } from "./EditorBottomBar";
 import Preview from "./Preview";
-import { ScrollSync, ScrollSyncPane } from "react-scroll-sync";
 
-function DocumentView() {
+const DocumentView = () => {
```

Comment:
Is there any reason to use an arrow function?
For consistency with other components, the `function` keyword would be better.